packages/framework/src/Model/ProductVideo/ProductVideoFacade.php

```diff
@@ -88,7 +88,7 @@ protected function persistVideoTranslations(
         foreach ($videoDataToCreate->videoTokenDescriptions as $descriptionLocale => $descriptionValue) {
             $productVideoTranslation = new ProductVideoTranslations();
             $productVideoTranslation->setLocale($descriptionLocale);
-            $productVideoTranslation->setDescription($descriptionValue ?? '');
+            $productVideoTranslation->setDescription($descriptionValue);
```

Comment:
in this PR we fix data, but it's possible to set the empty string again, or? Shouldn't we use `\Shopsys\FrameworkBundle\Component\String\TransformString::getTrimmedStringOrNullOnEmpty()`?